languages/CookieConsent/cs.ini

```diff
@@ -0,0 +1,32 @@
+Accept All Cookies = "Přijmout vše"
+Accept Only Essential Cookies = "Přijmout pouze nezbytné cookies"
+analytics_cookies_description_html = "<p>Analytické cookies umožňují sledovat souhrnné informace o&nbsp;návštěvnosti stránek.<br>Díky těmto informacím jsme schopni lépe přizpůsobit stránky uživatelům webu.</p><p>Jsou povoleny pouze na&nbsp;základě vašeho souhlasu.</p>"
+analytics_cookies_title_html = "Analytické cookies"
+category_description_html = "Cookies používané na těchto stránkách jsou rozděleny do několika kategorií podle účelu. Popis jednotlivých kategorií a možnost jejich přijetí nebo odmítnutí naleznete níže. <a href="%%siteRootAttr%%/Content/privacy" target="_blank">Informace o ochraně osobních údajů.</a>."
+cookie_description_cart_html = "Používá se k uložení obsahu košíku."
+cookie_description_consent_html = "Používá se k ukládání informací o tom, zda návštěvníci udělili souhlas nebo odmítli použití kategorií souborů cookie používaných na webu."
+cookie_description_language_html = "Používá se k uložení aktuálního jazyka uživatelského rozhraní."
+cookie_description_matomo_id_html = "Používá se pro statistiky webových stránek v aplikaci Matomo pro uložení identifikátoru uživatele."
+cookie_description_matomo_ref_html = "Používá se pro statistiky webových stránek v aplikaci Matomo pro uložení identifikátoru odkazu."
+cookie_description_matomo_ses_html = "Používá se pro statistiky webových stránek v aplikaci Matomo pro uložení identifikátoru relace."
+cookie_description_search_open_html = "Používá se k ukládání otevřených záznamů ve výsledcích vyhledávání."
+cookie_description_secure_key_html = "Používá se pro šifrování informací o relaci."
+cookie_description_session_html = "Používá se k udržování stavu relace uživatele."
+cookie_description_sidefacet_html = "Používá se k udržování stavu rozbalení/zabalení fasety."
+cookie_description_ui_html = "Slouží k uložení vybrané grafické šablony."
+cookie_settings_html = "Nastavit cookies"
+Description = "Popis"
+Domain = "Doména"
+essential_cookies_description_html = "<p>Funkční cookies jsou zapotřebí k zajištění základních funkcí webu.</p><p>Vzhledem k jejich podstatě a účelu jsou povoleny vždy a souhlas s jejich použitím není vyžadován.</p>"
```

Comment:
"<p>Funkční cookies jsou zapotřebí k zajištění základních funkcí webu.</p><p>Vzhledem k jejich významu a účelu jsou povoleny vždy a souhlas s jejich použitím není vyžadován.</p>"

Reply:
Fixed